sass/routes/_settings.scss

```diff
@@ -356,13 +356,18 @@
                     .theme-elements__body {
                         @include border-radius(0 0 3px 3px);
                         @include legacy-pie-clearfix;
-                        @include alpha-property(background-color, $white, .05);
-                        display: none;
+                        @include alpha-property(background-color, $white, 0.05);
+
                         margin: 0 20px 0 0;
-                        padding: 24px 0 0 24px;
+                        padding: 0 0 0 24px;
+                        transition: all 0.5s ease-in-out;
+                        max-height: 0;
+                        overflow-y: hidden;
 
                         &.open {
-                            display: block;
+                            max-height: 1000px;
```

Comment:
This appears to cause an issue with the theme selector in mobile view (`@media`/`max-width: 768px`)

![2020-09-28 14 23 10](https://user-images.githubusercontent.com/11724372/94477388-70421300-0197-11eb-8562-da02c85cd566.gif)

Reply:
I was looking at that as well since the arbitrary max-height seemed odd to me. It seems to be needed for the animation to behave smoothly for some reason. I tried setting it to other values like `100%`, but the animation is only smooth when it's set to some arbitrary pixel value. As seen in your gif, we'll have to increase it a bit since that section becomes longer than 1000px when the screen is small enough.

Reply:
While animating max-height can work, IMHO I think it would be better to use `react-transition-group` hooks (that might look something [like this](https://github.com/calebroseland/calebroseland-com/blob/master/src/components/transitions/components/CssTransition.vue#L40-L86))

Otherwise, Account Settings is slated to get a visual overhaul, so we could maybe switch over to a flexbox layout during that? (https://css-tricks.com/using-css-transitions-auto-dimensions/#bonus-technique-flexbox)

@hmhealey do you have any thoughts on this?

Reply:
`react-transition-group` hooks would still need to use css transitions. As I understand it?
So I think I do not come around animating max-height. :/